### DISCUSSION TITLE
About using shadow dom

### DISCUSSION THREAD

**Discussion by guiseek** [nrwl/nx#9568] [General]
https://github.com/nrwl/nx/blob/0155372379516f5a63120fc6a4c849ca8e511e23/packages/web/src/generators/application/files/app/src/app/app.element.ts__tmpl__#L10

what do you think about using shadow dom in this app generator implementation, in the web package?

would be the "most correct" way of working with web components, or rather, the most recommended.

however this hampers the benefits brought by sass, is there any discussion about it? what is the opinion of the team at nrwl and the community around this issue?

ps: i tried to create a discussion but the send button was disabled

**Comment by AgentEnder** [comment]
@guiseek I went ahead and converted this one to a discussion, I'm not sure why it wouldn't have let you.
